### ISSUE TITLE
new module: VITESSCE/VITESSCE

### ISSUE THREAD

**Issue by FloWuenne** [nf-core/modules#5139]
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

**Comment by FloWuenne** [comment]
The idea for this PR is to add a wrapper for [VITESSCE](http://vitessce.io/) that will generate html pages with embedded components pointing to the data in object stores (such as AWS S3). These vitessce html pages could then be viewed by users either on their machines or potentially even directly in Seqera Platform.

**Comment by melonora** [comment]
This could be soon linked with SpatialData. Support of Vitessce for SpatialData will be extended next week during the Boston scverse hackathon.

**Comment by FloWuenne** [comment]
After starting to work on it, I believe Vitessce will likely need 2 modules:
1) Vitessce/prep
2) Vitessce/view
or something along those lines. The first module will have to prepare the file formats to be compatible with Vitessce, the second one would generate the static html pointing to the S3 bucket files (or other stored files that can be read by vitessce. Since most pipelines won't by default have file formats compatible with vitessce, I think this will be necessary. This could then become a Vitessce subworkflow. What do you think @melonora ?

**Comment by FloWuenne** [comment]
Yeah it would definitely be cool to get some input from the vitessce authors about what they envision is the best setup for nextflow modules to prepare + view common data types using vitessce, thanks!

**Comment by FloWuenne** [comment]
Closing this issue for now, as I am not going to be working on this in the near future!
